lib/rules/no-unknown-property.js

```diff
@@ -230,7 +237,12 @@ function getDOMPropertyNames(context) {
   const ALL_DOM_PROPERTY_NAMES = DOM_PROPERTY_NAMES_TWO_WORDS.concat(DOM_PROPERTY_NAMES_ONE_WORD);
   // this was removed in React v16.1+, see https://github.com/facebook/react/pull/10823
   if (!testReactVersion(context, '>= 16.1.0')) {
-    return ['allowTransparency'].concat(ALL_DOM_PROPERTY_NAMES);
+    return ALL_DOM_PROPERTY_NAMES.concat(['allowTransparency']);
```

Comment:
```suggestion
    return ALL_DOM_PROPERTY_NAMES.concat('allowTransparency');
```
what’s the reason to flip these?